sources/Cargo.lock

```diff
@@ -2661,6 +2661,7 @@ dependencies = [
 name = "netdog"
 version = "0.1.0"
 dependencies = [
+ "anyhow",
```

Comment:
Please stick with `snafu` for OS components. Also, I don't think this should be part of this commit, but that's irrelevant if it goes away.